api/v1/types/util.go

```diff
@@ -63,3 +69,23 @@ func (p *GetLayerStatsActiveAccountsParams) Validate() error {
 	}
 	return &InvalidParamFormatError{ParamName: "window_step_seconds", Err: fmt.Errorf("invalid value: %d", *p.WindowStepSeconds)}
 }
+
+// The autogenerated server performs type validation that the input string
+// conforms to either an oasis bech32-encoded address or an ethereum
+// address, but we double check just to be sure.
```

Comment:
This is a comment on why we have this func. It's a good extra, but please prepend it with a description of what this func does.

Also, semi-nit: Openapi specifies a regex pattern, but the autogenerated code ignores that pattern. So this func is _the_ validity check. Example: https://nexus.oasis.io/v1/sapphire/accounts/foobar

Reply:
Oh I thought openapi was doing input validation somewhere deep down; that's good to know. Updated with comment!